core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -114,6 +114,22 @@ class KafkaZkClient(zooKeeperClient: ZooKeeperClient, isSecure: Boolean) extends
   }
 
   /**
+   * Creates the delete topic znode.
+   * @param topicName topic name
+   * @throws KeeperException if there is an error while setting or creating the znode
+   */
+  def createDeleteTopicPath(topicName: String): Unit = {
+    createRecursive(DeleteTopicsZNode.path)
+    val createRequest = CreateRequest(DeleteTopicsTopicZNode.path(topicName), null,
+      acls(DeleteTopicsZNode.path), CreateMode.PERSISTENT)
+
+    val createResponse =  retryRequestUntilConnected(createRequest)
+    if (createResponse.resultCode != Code.OK) {
+      createResponse.resultException.foreach(e => throw e)
+    }
```

Comment:
Could this method just be createRecursive(DeleteTopicsTopicZNode.path(topicName))?